db/migrate/20160321222121_change_default_value_of_involved_in_gamification.rb

```diff
@@ -0,0 +1,9 @@
+class ChangeDefaultValueOfInvolvedInGamification < ActiveRecord::Migration
+  def up
+    change_column_default :users, :involved_in_gamification, false
```

Comment:
Pozor, migracie sa pustaju po sebe. cize najskor sa spusti `20160311200950_add_involved_in_gamification_to_users.rb`, ktora do tabulky `Users` prida stlpec `involved_in_gamification` a kedze je tam default `true`, tak sa vsetkym nastavi true. Ked sa v tejto migracii potom uz len zmeni default na `false`, tak existujucich pouzivatelov, ktori uz maju vsetci nastavene `true` sa to nijako nedotkne, ovplyvni to len tych, ktori vzniknu odteraz (prihlasia sa prvykrat a pod.).

Po spravnosti by tu este mala byt migracia, ktora vsetkym existujucim pouzivatelom nastavi `false`. Aj ked na produkcii je to uz nasadene a fixol som to rucne v DB, treba tu migraciu spravit, lebo ked sa to napriklad nasadi na iny stage, ked si to niekto spusti u seba, atd., zase bude vsade true.